client/finality-grandpa/rpc/src/error.rs

```diff
@@ -45,6 +46,7 @@ pub enum ErrorCode {
 	ProveFinality,
 }
 
+// TODO: remove (?)
```

Comment:
we need to add this to jsonrpsee, it's not possible to set application specific error codes.